browser/extensions/brave_component_extension_resource_manager.cc

```diff
@@ -4,11 +4,14 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/browser/extensions/brave_component_extension_resource_manager.h"
+#include "brave/components/brave_rewards/browser/buildflags/buildflags.h"
 
 #include "brave/components/brave_extension/grit/brave_extension_generated_map.h"
 #include "brave/components/brave_extension/grit/brave_extension_resources_map.h"
+#if BUILDFLAG(BRAVE_REWARDS_ENABLED)
```

Comment:
nit: There is an implicit convention in chromium about header inclusion in guard.
Usually they are positioned at the last of header including part like below.

```
#include "ui/views/widget/tooltip_manager.h"
#include "ui/views/widget/widget.h"

#if defined(OS_WIN)
#include "base/win/scoped_gdi_object.h"
#include "ui/native_theme/native_theme_win.h"
#endif
```

Reply:
Done.